core/modules/wildcard_match.cc

```diff
@@ -374,20 +383,21 @@ CommandResponse WildcardMatch::CommandDelete(
   return CommandSuccess();
 }
 
-CommandResponse WildcardMatch::CommandClear(const bess::pb::EmptyArg &) {
+void WildcardMatch::Clear() {
   for (auto &tuple : tuples_) {
     tuple.ht.Clear();
   }
+}
 
-  CommandResponse response;
-
+CommandResponse WildcardMatch::CommandClear(const bess::pb::EmptyArg &) {
+  WildcardMatch::Clear();
   return CommandSuccess();
 }
 
-CommandResponse WildcardMatch::CommandGetRules(const bess::pb::EmptyArg &) {
-  bess::pb::WildcardMatchCommandGetRulesResponse resp;
-  resp.set_default_gate(default_gate_);
-
+// This is left over from when there was a separate
+// WildcardMatchCommandGetRules; we could just inline this now.
+template <typename T>
+void WildcardMatch::GetFields(T &resp) {
```

Comment:
Not that I like this convention, but for consistency: the Google style guide states that in the argument list mutable objects should be given as a pointer, and non-const references are not allowed [(link)](http://google.github.io/styleguide/cppguide.html#Reference_Arguments).

Reply:
Ah, right, I'm still slowly absorbing the Google style guide.  Will fix.